faust_large_message_serializer/clients/storing_client.py

```diff
@@ -0,0 +1,56 @@
+from typing import Union
+from uuid import uuid4
+
+from loguru import logger
+
+from faust_large_message_serializer.blob_storage.blob_storage import BlobStorageClient
+from faust_large_message_serializer.utils.uri_parser import URIParser
+
+
+class StoringClient:
+
+    VALUE_PREFIX = "values"
+    KEY_PREFIX = "keys"
+    IS_BACKED = b"\x01"
+    IS_NOT_BACKED = b"\x00"
+
+    def __init__(self, client: BlobStorageClient, base_path: URIParser, max_size: int):
+        self._client = client
+        self._base_path = base_path
+        self._max_size = max_size
+
+    def store_bytes(self, topic: str, data: bytes, is_key: bool) -> Union[bytes, None]:
```

Comment:
Alternatively, you can just write:
```suggestion
    def store_bytes(self, topic: str, data: bytes, is_key: bool) -> Optional[bytes]:
```

Reply:
To match the if in line 23, the typing should be:
```suggestion
    def store_bytes(self, topic: str, data: Optional[bytes], is_key: bool) -> Union[bytes, None]:
```